wp-multi-network/includes/functions.php

```diff
@@ -456,33 +468,25 @@ function add_network( $args = array() ) {
 
 	// Bail if no user with this ID for site
 	if ( empty( $r['user_id'] ) || ! get_userdata( $r['user_id'] ) ) {
-		return new WP_Error( 'network_user', __( 'User does not exist.', 'wp-multi-network' ) );
+		return new WP_Error( 'network_user', __( 'User does not exist.', 'wp-multi-network' ), array( 'status' => 403 ) );
 	}
 
 	// Bail if no user with this ID for network
 	if ( empty( $r['network_admin_id'] ) || ! get_userdata( $r['network_admin_id'] ) ) {
-		return new WP_Error( 'network_super_admin', __( 'User does not exist.', 'wp-multi-network' ) );
+		return new WP_Error( 'network_super_admin', __( 'User does not exist.', 'wp-multi-network' ), array( 'status' => 403 ) );
```

Comment:
The error statuses should be removed from this function, they should instead be handled in the REST controller.

Furthermore I'd recommend to use 400 as status code for all of them, because in each case it simply means the request is invalid.